jdaviz/configs/default/plugins/viewers.py

```diff
@@ -164,7 +191,7 @@ def _apply_layer_defaults(self, layer_state):
             layer_state.add_callback('as_steps', self._show_uncertainty_changed)
 
     def _expected_subset_layer_default(self, layer_state):
-        if self.__class__.__name__ == 'CubevizImageView':
+        if self.__class__.__name__ in ('CubevizImageView', 'RampvizImageView'):
```

Comment:
could/should this now be an `isinstance` on the base Cube class?  (I can't remember if this was originally checking the name to avoid a recursion error or something else - but maybe it can be avoided now?)